.jshintignore

```diff
@@ -59,6 +59,7 @@ apps/system/js/uuid.js
 apps/camera/bower_components/**
 apps/bluetooth/js/vendor/alameda.js
 apps/bluetooth/js/settings.js
+apps/homescreen-ng/bower_components/**
```

Comment:
Is there any harm in just naming this app `homescreen`? Not really a fan of `ng`, at some point it will not be `new`.